src/WindowTitle.ps1

```diff
@@ -58,3 +58,45 @@ function Set-WindowTitle {
         }
     }
 }
+
+function Set-TabTitle {
```

Comment:
If we change the setting to be a scriptblock, then this function becomes very similar to the Set-WindowTitle function above except that where it sets the title, it uses `$psise.CurrentPowerShellTab.DisplayName = …`